modules/apps/sharing/sharing-web/src/main/resources/META-INF/resources/shared_assets/view.jsp

```diff
@@ -106,22 +121,8 @@ viewSharedAssetsDisplayContext.populateResults(sharingEntriesSearchContainer);
 	</liferay-ui:search-container>
 </clay:container-fluid>
 
-<%
-PortletURL viewAssetTypeURL = PortletURLUtil.clone(currentURLObj, liferayPortletResponse);
-
-viewAssetTypeURL.setParameter("className", (String)null);
-
-PortletURL selectAssetTypeURL = viewSharedAssetsDisplayContext.getSelectAssetTypeURL();
-%>
-
-<liferay-frontend:component
-	componentId='<%= liferayPortletResponse.getNamespace() + "SharedAssets" %>'
-	context='<%=
-		HashMapBuilder.<String, Object>put(
-			"selectAssetTypeURL", selectAssetTypeURL.toString()
-		).put(
-			"viewAssetTypeURL", viewAssetTypeURL.toString()
-		).build()
-	%>'
-	module="SharedAssets.es"
-/>
\ No newline at end of file
+<aui:script>
+	Liferay.on('sharing:changed', () =>
+		Liferay.Portlet.refresh('#p_p_id<portlet:namespace />')
+	);
+</aui:script>
```

Comment:
Would it be better in a separate file?

Maybe with dispose method returned

```
return {
  dispose: () => Liferay.deatch(...
}
```

I read the previous discussion maybe in a new one or in the old SharedAssets.es.js, not in the SharedAssetsManagementToolbarPropsTransformer.js.

Reply:
Ideally we don't want to keep `js` code in `jsp`s, but **introducing** new js code in a `jsp` is definitely something we must not do.

Reply:
@carloslancha do you agree with @boton 's proposal? Any other suggestion @julien @markocicos?

Reply:
I'd create a `SharedAssets` component to put it there, yup

Reply:
done in https://github.com/liferay-lima/liferay-portal/pull/1548/commits/89ccb0b567f0bf783f062f0580525810d5e0fbcb

Reply:
LGTM, Ready to ship.
What do you think @carloslancha?